### PR TITLE
Polish the package (Jest, CircleCI, TypeScript, code coverage)

### PR DESCRIPTION
**Pull request by maticzav**
- [x] CircleCI configuration
- [ ] Code coverage (100%)
- [ ] Jest tests
- [ ] TypeScript type-safety

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,6 @@
-/dist/
-/node_modules/
+node_modules
+dist
+coverage
+
+*.log*
+.DS_Store
```
Comment (LitoMore) [inline .gitignore]: You could add `.DS_Store` to your global ignore file. See here: https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Comment (LitoMore) [reply .gitignore]: I think there should be a newline at the end of this file.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+version: 2.0
+jobs:
+  test:
+    docker:
+      - image: 'circleci/node:11.3.0'
```
Comment (LitoMore) [inline .circleci/config.yml]: We should use `node:8` and `node:10` for CI.

File: .gitignore (hunk 1 above)
Comment (maticzav) [reply .gitignore]: I'd rather include the `.DS_Store` to make sure anyone else who submits a PR doesn't accidentally commit it in the code.

Comment (LitoMore) [reply .gitignore]: > I'd rather include the .DS_Store to make sure anyone else who submits a PR doesn't accidentally commit it in the code.

Make sense.

**Pull request closed without merging**